### PR TITLE
Feature show profile button

### LINKED ISSUE #35: As a user, I would like to see if I am a user or a admin, so that I know my privilege level and abilities

### PR DESCRIPTION
**Pull request by Pirundy**
Added the profile button in the navbar and also the button in the admin page buttons with a new profile page that may be removed in the future.

### CODE REVIEW

File: views/pages/profile/index.ejs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <div class="card-body">
+    <h2>Name:</h2>
+  </div>
+  <hr />
+  <div class="card-body">
+    <h2>User ID:</h2>
+  </div>
+  <hr />
+  <div class="card-body">
+    <h2>Email:</h2>
```
Comment (Jake-Derrick) [inline views/pages/profile/index.ejs]: Could add in <%= email %> since we have that data

**Review by Jake-Derrick — APPROVED** [review]
Looks good 👍

File: views/pages/profile/index.ejs (hunk 1 above)
Comment (Pirundy) [reply views/pages/profile/index.ejs]: This was added from your recommendation in commit 3c6e154 thanks!

**Review by presmiks18 — APPROVED** [review]
This is looking good to me, good job.

**Pull request merged**